api/src/main/resources/messages.properties

```diff
@@ -1 +1,3 @@
 ${project.parent.artifactId}.title=Location based access control module
+
+locationBasedAccess.patientDashboard.headerTitle=Patient Location
```

Comment:
The message key should tell that this is a patient location. For the module could display more than just location, on the dashboard. How about? ${project.parent.artifactId}.patientLocation

Reply:
Since I couldn't access this from the JSP page, I just used "locationbasedaccess.patientLocation". Please let me know if there are any changes needed.